### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by kenzyop**
Adding an issue template. It is the standard bug template minus the Desktop/Smartphone sections.

### CODE REVIEW

**Review by rocpoon — APPROVED** [review]
Looks good

**Review by rocpoon — APPROVED** [review]
Looks even better now, awesome

**Review by SoJexy — APPROVED** [review]
@kenzyop Looks good to me 👍🏻

File: .github/ISSUE_TEMPLATE/issue-report.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+name: Issue report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```
Comment (AndyHsiungDev) [inline .github/ISSUE_TEMPLATE/issue-report.md:11]: Is this supposed to be different from a bug report? Perhaps this template should be a "Feature Request" instead.

**Review by AndyHsiungDev — CHANGES_REQUESTED** [review]

**Review by AndyHsiungDev — APPROVED** [review]

**Pull request merged**